packages/babel-plugin-transform-es2015-modules-systemjs/src/index.js

```diff
@@ -68,7 +72,17 @@ export default function ({ types: t }) {
   };
 
   return {
+    inherits: syntax,
```

Comment:
Should the systemjs-module transform really depend on the syntax? wouldn't it be up to the user if he want's to support this syntax or not. Because this means the preset-es2015 with systemjs would active the syntax for a stage proposal.